operator/apis/config/v1/doc.go

```diff
@@ -18,3 +18,7 @@ var (
 	// AddToScheme adds the types in this group-version to the given scheme.
 	AddToScheme = SchemeBuilder.AddToScheme
 )
+
+func init() {
+	SchemeBuilder.Register(&ControllerManagerConfiguration{})
+}
```

Comment:
I don't think this is necessary. We use `ProjectConfig` as kind for our configuration file, which is registered. And it does also not seem to be necessary for generating the Markdown files in `doc/`.

```suggestion
```